src/test/elapsed_time.py

```diff
@@ -0,0 +1,17 @@
+from util import *
+
+# Test that the elapsed-time GDB command returns a time >= 1.0 at a breakpoint
+# after sleep(1);
+
+
+send_gdb('b breakpoint')
+expect_gdb('Breakpoint 1')
+
+send_gdb('elapsed-time')
+
+# first group captures numbers >= 1.0
+# second group captures numbers >= 10.0
+expect_gdb('^(?:[1-9]\.\d*|\d\d+\.\d*)$')
```

Comment:
Would be a bit cleaner to capture the floating point value, convert to float, and then compare it to 1.